src/misc/Comment.ts

```diff
@@ -0,0 +1,32 @@
+import { IContainer } from "../Component";
+import IMain from "../IMain";
+import Inert from "../Inert";
+
+export interface ICommentOpts {
+    text: string;
+}
+
+/**
+ * A component containing free text.
+ */
+export default class Comment extends Inert {
+    public static componentName: string = "Comment";
+    public static componentTag: string = "misc";
+    protected static optUpdateHandlers = {
+        text: "updateText",
+    };
+    protected static defaultOptions: ICommentOpts = {
+        text: "",
+    };
+
+    protected opts: ICommentOpts;
+    private text: string;
+
+    constructor(main: IMain, parent: IContainer, opts: any) {
+        super(main, parent, opts);
+    }
+
+    public updateText() {
```

Comment:
We don't really need this update handler or the `text` private property. The private `text` property is not used anywhere inside this class and it's not exposed to anyone so it has no purpose. The `text` option is however already maintained in the `opts` property. When Webvs clients update them with the [`set`](https://github.com/azeem/webvs/blob/master/src/Model.ts#L53) method on a component it is [automatically updated](https://github.com/azeem/webvs/blob/master/src/Component.ts#L241).

The update handlers are a way for the component to react to changes in opts and update internal state, re-compile shaders etc.